### DISCUSSION TITLE
[Connector Request]

### DISCUSSION THREAD

**Discussion by okxwallet** [wagmi-dev/references#160] [Connector request]
### What **novel use-case** does the Connector provide?

Hi wagmi team,

I want to integrate our browser extension OKX Wallet (https://www.okx.com/web3) Following the breadcrumbs, I stumbled here. Just wanted to confirm that what I am planning is correct before beginning.

We are multichain wallet and have other providers as well with their on SDKs as can be seen [here](https://www.okx.com/web3-docs/extension/introduction.html).

However, for EVM support I assume the below steps should suffice:

I need to add an isOKXWallet property to InjectedProviderFlags and getInjectNames.ts, and update the equivalent test files.

Please let me know if I am missing anything, and thank you for your help!

### Are the Connector's integrations production-ready and generally available?

OKX extension wallet(https://chrome.google.com/webstore/detail/okx-wallet/mcohilncbfahbmgdjkbpemcciiolgcge)

### Are you committed to actively mainting the Connector?

- [X] Yes, my team is or I am committed to actively maintaining the Connector.

**Comment by tmm** [comment]
> I need to add an isOKXWallet property to InjectedProviderFlags and getInjectNames.ts, and update the equivalent test files.

Sounds good!
